_posts/2020-06-01-deleting-list-of-builds.md

```diff
@@ -4,8 +4,16 @@ author: Dominik Tureček
 layout: post
 ---
 
-Since the latest release, we are now able to delete a group of builds specified by their IDs with a single command in the following style:
+Previously, when you wanted to delete lots of build in the Copr you have to do something like:
```

Comment:
Previously, when you wanted to delete lots of build in the Copr, you had to do something like: